requirements.txt

```diff
@@ -1,5 +1,6 @@
-pyflwor_ext==1.2
-rospkg==1.1.4
-bonsai_code==0.4.5
-setuptools==20.7.0
-PyYAML==3.13
+pyflwor_ext>=1.2
```

Comment:
I realise it was already there before, but isn't the correct name `pyflwor-ext` instead?

Or does `pip` check underscores and dashes automatically?

Reply:
The original file was automatically generated, I think. And it seems [from this SO answer](https://stackoverflow.com/a/19131777) that `pip` replaces the underscore automatically.